pages/post/[post].tsx

```diff
@@ -0,0 +1,144 @@
+import { useRouter } from 'next/router'
+import { fetchEntries } from "../../utils/contentfulPosts";
+import Link from 'next/link'
+
+
+interface HomeProps {
+    title: string
+    category: string
+    publishedDate: string
+    imageAlt: string
+    imageUrl: string
+    slug: string
+    author: string
+    body: string
+  }
+
+export default function Home({ content }: HomeProps){
```

Comment:
This is causing the error on the Netlify deployment. The attribute you are referencing in the prop destructuring is not defined in the HomeProps interface. Looking at how you are using **content**, you should then avoid using destructuring, since **content** will be an object containing all the props.